scripts/istio-images-mirror

```diff
@@ -25,8 +25,14 @@ library/nginx rancher/nginx 1.17.4-alpine
 jenkins/jnlp-slave rancher/jenkins-jnlp-slave 3.40-1
 thanosio/thanos rancher/thanos v0.10.1
 quay.io/jetstack/cert-manager-controller rancher/jetstack-cert-manager-controller v0.8.1
-quay.io/calico/node rancher/calico-node v3.10.2
-quay.io/calico/cni rancher/calico-cni v3.10.2
-quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.10.2
-quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.10.2
-us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-external-dns v0.6.0
\ No newline at end of file
+quay.io/calico/node rancher/calico-node v3.12.0
+quay.io/calico/cni rancher/calico-cni v3.12.0
+quay.io/calico/ctl rancher/calico-ctl v3.12.0
+quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.12.0
+quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.12.0
+us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-external-dns v0.6.0
+gcr.io/google_containers/k8s-dns-kube-dns rancher/k8s-dns-kube-dns 1.15.9
```

Comment:
I don't know if this mirror image does mult-arch builds and this image seems to have multi-arch images? 
https://hub.docker.com/repository/docker/rancher/k8s-dns-kube-dns/tags?page=1

Reply:
yes this appears to be a problem. I don't think image mirror scripts supports tagging with manifest. I will remove these image then.